### DISCUSSION TITLE
Running GUI apps inside container

### DISCUSSION THREAD

**Discussion by Joel-N99** [Dvlv/BoxBuddyRS#148] [General]
Hi,
I am contemplating moving over to atomic distros like bazzite and ublue aurora and so I am giving boxbuddy a try to make sure everything i need works in it. so far I was able to install chromium and run it and it seems to be running fine. But online I read that docker containers have a hard time running GUI apps and one has to tinker with the configs to get it running smoothly. What is boxbuddy doing underneath that is seemingly making the browser run so smooth so far and are there any problems in this system that I should be aware of before jumping into atomic distros?
I will mostly run browsers, vscode and maybe in future webex/zoom meeting apps.
Also I see that my entire home directory has been exposed into the container automatically which is what I was trying to do manually with docker-cli so this is great and means that the apps i will run will be able to access their config files, bookmarks etc

p.s: the "run" and "add to menu" buttons in the applications menu looks a bit odd. maybe decrease their sizes and remove that white line from the middle? 

Thanks!

**Comment by Dvlv** [comment]
Hello,
These are questions for Distrobox: https://github.com/89luca89/distrobox
BoxBuddy is just a GUI which calls distrobox commands as subprocesses

**Reply by Joel-N99** [reply]
Alright 👍🏽
